src/drive/mobile/containers/MediaBackupProgression.spec.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import configureStore from 'redux-mock-store'
+import { shallow } from 'enzyme'
+
+const render = () => {
+  const MediaBackupProgression = require('./MediaBackupProgression').default
```

Comment:
Out of curiosity, why are you using `require` and not `import` here?

Reply:
Parce que je re-mocke les sélecteurs pour chaque test, mais comme ils sont importés par le composant, je dois le `require` à chaque fois pour que les sélecteurs soient correctement mockés